app/scripts/controllers/transactions/tx-gas-utils.js

```diff
@@ -76,8 +76,27 @@ export default class TxGasUtil {
     delete txParams.maxFeePerGas;
     delete txParams.maxPriorityFeePerGas;
 
-    // estimate tx gas requirements
-    return await this.query.estimateGas(txParams);
+    // dont use ethjs-query here because it will blow up about accessList
+    if (txParams.accessList) {
+      return await new Promise((resolve, reject) => {
+        this.query.rpc.currentProvider.sendAsync(
```

Comment:
Should we just update ethjs-query? What will be the difference by using the provider directly vs. using ethjs-query?

Reply:
Pr for updating it

https://github.com/ethjs/ethjs-schema/pull/13